bluepyentity/default_params.yaml

```diff
@@ -0,0 +1,42 @@
+# Currently, mostly a placeholder for possible default values
+
+# CIRCUIT
+
+DetailedCircuit:
+  type: DetailedCircuit
+
+# SIMULATION
+
+Simulation:
+  type: Simulation
+
+# SIMULATION CAMPAIGN
+
+SimulationCampaignConfiguration:
+  type: SimulationCampaignConfiguration
+
+# ANALYSIS
+
+AnalysisReport: # the actual analysis report
+  type: AnalysisReport
+
+DetailedCircuitValidationReport: # same as AnalysisReport
+  type: DetailedCircuitValidationReport
+
+# SUBJECT
+
+Subject:
```

Comment:
Looks like subject is like BrainLocation:
```python
  "subject": {
    "@type": "Subject",
    "species": {
      "@id": "http://purl.obolibrary.org/obo/NCBITaxon_10090",
      "label": "Mus musculus"
    }
  }
}
I was looking at:
https://bbp.epfl.ch/nexus/web/bbp/atlasdatasetrelease/resources/https%3A%2F%2Fbbp.epfl.ch%2Fneurosciencegraph%2Fdata%2Fcellcompositions%2Fca364248-581b-41b7-bf80-f7383cab5e48
```

Reply:
I would opt for retrieving the label from the resource id to minimize human error..

Reply:
I believe it has some value to be able to set it, but 99.9% of the time, the default will be used yeah.